api/types/container/secret.go

```diff
@@ -0,0 +1,12 @@
+package container
+
+import "os"
+
+type ContainerSecret struct {
+	Name   string
+	Target string
+	Data   []byte
+	Uid    int
```

Comment:
ok, but just note that `api/types/plugin.go` uses `UID`
https://github.com/docker/docker/blob/master/api/types/plugin.go#L162